codejs.js

```diff
@@ -1,91 +1,86 @@
-
-
-const cabanas = document.querySelectorAll (".cabana");
-const poput = document.querySelector('.back-Ground');
-const Xboton = document.querySelector('.Xboton');
+const cabanas = document.querySelectorAll(".cabana");
+const poput = document.querySelector(".back-Ground");
+const Xboton = document.querySelector(".Xboton");
 
 let currentCabana = undefined;
 
-for (let i=0; i < cabanas.length; i++) {
-    const aCabana = cabanas[i];
-    aCabana.addEventListener('click', () => {
-        currentCabana = i;
-        poput.classList.remove('hide');
-    });
-};
-if (Xboton){
-    Xboton.addEventListener('click', () => {
-        poput.classList.add('hide');
-    });  
+for (let i = 0; i < cabanas.length; i++) {
+  const aCabana = cabanas[i];
+  aCabana.addEventListener("click", () => {
+    currentCabana = i;
+    poput.classList.remove("hide");
+  });
```

Comment:
nit: strings in JS should be single quote, not double.
```suggestion
  aCabana.addEventListener('click', () => {
    currentCabana = i;
    poput.classList.remove('hide');
  });
```